docs/get_started/getting_started_riscv_cmake.md

```diff
@@ -0,0 +1,123 @@
+# Getting Started on RISC-V with CMake
+
+<!--
+Notes to those updating this guide:
+
+    * This document should be __simple__ and cover essential items only.
+      Notes for optional components should go in separate files.
+-->
+
+This guide walks through cross-compiling IREE core runtime towards the RISC-V Linux platform. Cross-compiling IREE compilers towards RISC-V is not supported at the moment.
+
+Cross-compilation involves both a *host* platform and a *target* platform. One
+invokes compiler toolchains on the host platform to generate libraries and
+executables that can be run on the target platform.
+
+## Prerequisites
+
+### Set up host development environment
+
+The host platform should have been set up for developing IREE. Right now Linux
+and Windows are supported. Please make sure you have followed the steps for
+[Linux](./getting_started_linux_cmake.md) or
+[Windows](./getting_started_windows_cmake.md).
+
+### Install RISC-V Tools
+
+Execute the following script to download RISC-V toolchain and QEMU:
+```shell
+# In IREE source root
+$ ./scripts/riscv_bootstrap.sh
+```
```

Comment:
I personally prefer adding a blank line before and after the snippet.

Reply:
OK